tmt/export.py

```diff
@@ -513,15 +604,173 @@ def export_to_nitrate(test):
+    if not dry_mode:
+        polarion_case.description = description
+    echo(style('description: ', fg='green') + description)
+
+    # Automation
+    if test.node.get('extra-task'):
+        automation_script = test.node.get('extra-task')
+        automation_script += ' ' + test.fmf_id['url']
+    else:
+        automation_script = test.fmf_id['url']
+    if not dry_mode:
+        polarion_case.caseautomation = 'automated'
+        polarion_case.automation_script = automation_script
+    echo(style('script: ', fg='green') + automation_script)
+
+    # Components
+    if not dry_mode:
+        polarion_case.caselevel = 'component'
+        polarion_case.testtype = 'functional'
+        if test.component:
+            polarion_case.casecomponent = [test.component[0]]
+    echo(style('components: ', fg='green') + ' '.join(test.component))
+
+    # Tags and Importance
+    if not dry_mode:
+        if test.tier is not None:
+            if int(test.tier) <= 1:
+                polarion_case.caseimportance = 'high'
+            elif int(test.tier) == 2:
+                polarion_case.caseimportance = 'medium'
+            else:
+                polarion_case.caseimportance = 'low'
+            test.tag.append(f"Tier{test.tier}")
+        else:
+            polarion_case.caseimportance = 'medium'
+
+    test.tag.append('fmf-export')
+    if not dry_mode:
+        polarion_case.tags = ' '.join(test.tag)
+    echo(style('tags: ', fg='green') + ' '.join(set(test.tag)))
+
+    # Default tester
+    if test.contact:
+        # Need to pick one value, so picking the first contact
+        email_address = email.utils.parseaddr(test.contact[0])[1]
+        login_name = email_address[:email_address.find('@')]
         try:
             if not dry_mode:
-                bz_set_coverage(
-                    bz_instance, verifies_bug_ids, int(
-                        nitrate_case.id))
-            echo(style("Linked to Bugzilla: {}.".format(", ".join(
-                [f"BZ#{bz_id}" for bz_id in verifies_bug_ids])), fg='magenta'))
+                polarion_case.add_assignee(login_name)
+            echo(style('default tester: ', fg='green') + login_name)
+        except PolarionException as err:
+            log.debug(err)
+
+    # Status
+    if not dry_mode:
+        if test.enabled:
+            polarion_case.status = 'approved'
+        else:
+            polarion_case.status = 'inactive'
+    echo(style('enabled: ', fg='green') + str(test.enabled))
+
+    echo(style("Append the Polarion test case link.", fg='green'))
+    if not dry_mode:
+        with test.node as data:
+            server_url = str(polarion_case._session._server.url)
+            add_link(
+                f'{server_url}{"" if server_url.endswith("/") else "/"}'
+                f'#/project/{polarion_case.project_id}/workitem?id='
+                f'{str(polarion_case.work_item_id)}',
+                data, system=SYSTEM_OTHER, type_='implements')
```

Comment:
The `implements` relation is intended for application code implementing a story (feature/requirement). Shall we rather use `relates` here? Not sure. @lukaszachy, what do you think?

Reply:
from my understanding of your description of this link type it seemed appropriate, but it doesn't really matter which type it is to me

Reply:
from my understanding of your description of this link type it seemed appropriate, but it doesn't really matter which type it is to me

Reply:
@psss IMHO `implements` is correct. This should be relation saying "polarion case equals tmt test" and afaik closest to that is 'implements'. We don't do that for nitrate (there is no "link" per se, just [fmf] section).
You can think of Polarion case as a story describing what has to be tested and such test is done (implemented) in the tmt case.

Reply:
Makes sense. After discussion `implements` sounds as the best choice.